module/x/gravity/abci.go

```diff
@@ -308,8 +308,7 @@ func cleanupTimedOutContractCallTxs(ctx sdk.Context, k keeper.Keeper) {
 		cctx, _ := otx.(*types.ContractCallTx)
 		if cctx.Timeout < ethereumHeight {
 			// we keep the data around for slashing in case the timeout was due to a lack of signatures
-			k.SetCompletedOutgoingTx(ctx, cctx)
-			k.DeleteOutgoingTx(ctx, cctx.GetStoreIndex())
+			k.Complete(ctx, cctx)
```

Comment:
`Complete` is a bit of a vague name if we're using this to bundle together things here. Perhaps `CompleteOutgoingTx`?